master/reference/public-cloud/gce.md

```diff
@@ -11,13 +11,15 @@ options for doing this depending on your deployment.
 
 Calico requires the following firewall rules to function in GCE.
 
-| Description      | Protocol | Port Range |
-|:-----------------|:---------|:-----------|
-| BGP              | TCP      | 179        |
-| \*IPIP           | 4        | all        |
+| Description       | Protocol | Port Range |
+|:------------------|:---------|:-----------|
+| `BGP`             | TCP      | 179        |
+| `\*IPIP`*         | 4        | all        |
+
+
+\* The `\*IPIP` rule is required only when using Calico with IPIP encapsulation. Keep reading 
```

Comment:
Here it looks like the `\*` in `\*IPIP` is an intentional part of the content - and I am not familiar with this detail, but happy to assume that that is correct.  But I would guess that the `\*` at the start of the line is not needed - unless it is a special formatting marker.

Reply:
I'm glad that you mentioned this. We had changed the AWS docs around this and now I am modifying the GCE docs to match. For some reason, in the original note (also in AWS), there was a `\* that preceded `IPIP` and I didn't know what to make of it. We have decided that these characters were included erroneously.

But, as noted there, the `\* below will become just `*` in the rendered content as the `\` escapes the asterisk that would otherwise get rendered as a bullet.